### PR TITLE
[docs] Update devupdate with FA6 info

### PR DESCRIPTION
**Pull request by sarjona**
This issue is to document changes done in [MDL-82497](https://tracker.moodle.org/browse/MDL-82497), [MDL-82211](https://tracker.moodle.org/browse/MDL-82211) and [MDL-82210](https://tracker.moodle.org/browse/MDL-82210).

### CODE REVIEW

File: docs/devupdate.md [hunk 1]
```diff
@@ -150,3 +150,49 @@ class core_renderer extends \core_renderer {
 Some of the Bootstrap 4 classes will be deprecated or dropped in its version 5. To prepare for this, some of the current Bootstrap 4 classes usages have been replaced with version 5 compatible classes using a "bridge". This will help us to upgrade to Bootstrap 5 in the future.
 
 See more information in [Bootstrap 5 migration](./guides/bs5migration/index.md).
+
+### Support FontAwesome families
+
+<Since version="4.5" issueNumber="MDL-82210" />
+
+Upon upgrading Font Awesome (FA) from version 4 to 6, the solid family was selected by default. However, FA6 includes additional families such as regular and brands. Support for these families has now been integrated, allowing icons defined with icon_system::FONTAWESOME to use them.
```
Comment (kabalin) [inline docs/devupdate.md:158]:
```suggestion
Upon upgrading Font Awesome (FA) from version 4 to 6, the solid family was selected by default. However, FA6 includes additional families such as regular and brands. Support for these families has now been integrated, allowing icons defined with `icon_system::FONTAWESOME` to use them.
```

File: docs/devupdate.md [hunk 2]
```diff
@@ -150,3 +150,49 @@ class core_renderer extends \core_renderer {
...
+Icons can add the FontAwesome family (fa-regular, fa-brands, fa-solid) near the icon name to display it using this styling:
```
Comment (kabalin) [inline docs/devupdate.md:159]:
```suggestion
Icons can add the FontAwesome family (`fa-regular`, `fa-brands`, `fa-solid`) near the icon name to display it using this styling:
```

**Review by kabalin — COMMENTED** [review]
Added a couple of formatting suggestions.

File: docs/devupdate.md (hunk 2 above)
Comment (sarjona) [reply docs/devupdate.md:159]: Done too :-)

**Comment by sarjona** [comment]
Hi @kabalin!
Thanks a lot for reviewing this. I updated the patch with the changes (well spotted!) and I rebased it too :-)

**Review by aanabit — APPROVED** [review]
The changes seems OK to me

**Pull request merged**